pallets/ajuna-affiliate/src/tests.rs

```diff
@@ -0,0 +1,327 @@
+use crate::{mock::*, *};
+use frame_support::{assert_noop, assert_ok};
+use sp_runtime::bounded_vec;
+
+#[test]
+fn test_extrinsic_id() {
+	use frame_support::traits::GetCallIndex;
+
+	ExtBuilder::default().build().execute_with(|| {
+		use frame_support::traits::PalletInfoAccess;
+		let balance_index = Balances::index();
+		println!("pallet index is {:?}", balance_index);
+		let call_index =
+			pallet::Call::<Test>::add_rule_to { extrinsic_id: (0, 0), rule: 0 }.get_call_index();
+		println!("call index is {:?}", call_index);
```

Comment:
Don't we want to assert the id here?

Reply:
This test was just for having an idea on how would a pallet call the trait methods. It will be removed later